ServantPersistent/src/Models.hs

```diff
@@ -0,0 +1,72 @@
+{-# LANGUAGE DataKinds       #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE TypeOperators   #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE TypeFamilyDependencies #-}
+{-# LANGUAGE TypeInType #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+module Models
+    where
+
+import Data.Aeson
+import Data.Aeson.TH
+import Database.Persist.Sql
+import Database.Persist.TH
+import GHC.Generics
+import Data.Time.Clock
+import Data.Text
+import Types
+import Control.Monad.IO.Class
+import Control.Monad.Reader
+import Control.Lens
+import Data.ByteString
+import Data.Proxy
+
+share [mkPersist sqlSettings { mpsGenerateLenses = True }, mkMigrate "migrateAll"] [persistLowerCase|
+DBTenant json
+    name Text
+    backofficeDomain Text
+    ownerId DBUserId Maybe
+    status TenantStatus
+    createdAt UTCTime
+    updatedAt UTCTime
+    UniqueTenant name backofficeDomain
+
+DBUser
+    firstName Text
+    lastName Text
+    tenantID DBTenantId
+    username Text
+    password ByteString
+    status UserStatus
+    createdAt UTCTime
+    updatedAt UTCTime
+    UniqueUsername username
+|]
+
+
+instance HasTimestamp DBTenant where
```

Comment:
Why do you need to do this? Can't classy lenses take care of this automatically?